docs/auth-provider-setup/verifiers.mdx

```diff
@@ -81,6 +81,26 @@ details.
 The list above is only comprehensive to some of the logins our system can support. If you'd like support for a particular login system, send your
 query to hello@web3auth.io.
 
+## Edit Verifier
+
+If you have created a verifier you'll see it under your personal verifiers.
```

Comment:
Are we addressing the team verifiers here?

Reply:
@ihsraham

Reply:
Resolved